design-doc-template.adoc

```diff
@@ -37,3 +37,11 @@
 === Non-Requirements
 
 == Test Plan
+
+== Implementation Plan
```

Comment:
Minor detail, but should this be above the test plan? It's not a huge deal though.

Reply:
My 2 cents: It's a less significant section than a test plan and often (aka usually) won't even be filled out, so it shouldn't be put in a higher priority section.

Reply:
That makes sense. I'll retract my comment :)

Reply:
I agree it should come before the test plan, not because of priority but because the test plan will likely reference the implementation.  For example "Implement xyz in module A" could be in the implementation plan, whereas "Create unit tests for implementation of xyz in module A" would refer to it.  It might be confusing to refer to something that comes later in the document.

Reply:
@dmlloyd @jamezp @bstansberry How about I move it up and comment out the header as well? Then people who want to use it can uncomment and fill it in

Reply:
Sounds good.